### PR TITLE
Update Gradle and Kotlin versions

### PR DESCRIPTION
**Pull request by abhijeet-nkt**
Kotlin: 1.3.50 => 1.3.61
Gradle: 3.4.1 => 3.5.3

### CODE REVIEW

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 1]
```diff
@@ -107,10 +107,6 @@ class SlideToActView @JvmOverloads constructor (
 
     /** Duration of the complete and reset animation (in milliseconds). */
     var animDuration: Long = 300
-        set(value) {
-            field = value
-            mAnimDuration = value
-        }
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: Your diff is dirty. Can you please rebase on top of master?

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 2]
```diff
@@ -193,8 +189,6 @@ class SlideToActView @JvmOverloads constructor (
     /** Paint used for inner elements */
     private val mInnerPaint: Paint = Paint(Paint.ANTI_ALIAS_FLAG)
 
-    private var mAnimDuration: Long = 300
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: ditto

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 3]
```diff
@@ -286,7 +280,7 @@ class SlideToActView @JvmOverloads constructor (
             isReversed = layoutAttrs.getBoolean(R.styleable.SlideToActView_slider_reversed, false)
             isRotateIcon = layoutAttrs.getBoolean(R.styleable.SlideToActView_rotate_icon, true)
             isAnimateCompletion = layoutAttrs.getBoolean(R.styleable.SlideToActView_animate_completion, true)
-            mAnimDuration = layoutAttrs.getInteger(R.styleable.SlideToActView_animation_duration, 300).toLong()
+            animDuration = layoutAttrs.getInteger(R.styleable.SlideToActView_animation_duration, 300).toLong()
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: ditto

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 4]
```diff
@@ -465,7 +459,7 @@ class SlideToActView @JvmOverloads constructor (
                     if ((mPosition > 0 && isLocked) || (mPosition > 0 && mPositionPerc < mGraceValue)) {
                         // Check for grace value
                         val positionAnimator = ValueAnimator.ofInt(mPosition, 0)
-                        positionAnimator.duration = mAnimDuration
+                        positionAnimator.duration = animDuration
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: ditto

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 5]
```diff
@@ -593,7 +587,7 @@ class SlideToActView @JvmOverloads constructor (
 
         animSet.playSequentially(*animators.toTypedArray())
 
-        animSet.duration = mAnimDuration
+        animSet.duration = animDuration
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: ditto

File: slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt [hunk 6]
```diff
@@ -697,7 +691,7 @@ class SlideToActView @JvmOverloads constructor (
             animSet.playSequentially(positionAnimator)
         }
 
-        animSet.duration = mAnimDuration
+        animSet.duration = animDuration
```
Comment (cortinico) [inline slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt]: ditto

**Comment by abhijeet-nkt** [comment]
Can you please show how to "rebase"?
I'm new to Git and this literally my second pulp request 😬

**Pull request closed without merging**